pkg/svcat/kube/kube.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package kube
 
 import (
+	_ "k8s.io/client-go/plugin/pkg/client/auth" // Load all client auth plugins for gcp, azure, etc
```

Comment:
I still hate inits, is there a function we can explicitly call to opt-in instead?

Reply:
There isn't. The init functions for the plugins all rely on unexported functions.